test/functional/apps/management/_scripted_fields.ts

```diff
@@ -240,10 +255,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
 
       it('should see scripted field value in Discover', async function () {
-        const fromTime = 'Sep 17, 2015 @ 06:31:44.000';
-        const toTime = 'Sep 18, 2015 @ 18:31:44.000';
         await PageObjects.common.navigateToApp('discover');
-        await PageObjects.timePicker.setAbsoluteRange(fromTime, toTime);
+        // await PageObjects.timePicker.setAbsoluteRange(fromTime, toTime);
```

Comment:
This looks like leftover code.

Also, I think you could just setTime once in the outermost describe's before method and unset it just once in the outermost after method instead of multiple times.  The test which only go into index patterns don't care what it's set to.  But up to you.